grpc/pom.xml

```diff
@@ -52,6 +52,10 @@
       <version>${grpc.version}</version>
     </dependency>
     <!-- Other -->
+    <dependency>
```

Comment:
Has direct usage, adding dependency.

Reply:
Here is the same, some gRPC dep was introducing guava and we now overwrote the version most likely

Reply:
Changed the approach to avoid this issue.